source/blood/src/menu.cpp

```diff
@@ -39,6 +39,10 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #include "screen.h"
 #include "sound.h"
 #include "view.h"
+#ifdef _WIN32 // required for MME midi device selection
+#include "../src/driver_winmm.h"
+#include "../src/midi.h"
+#endif
```

Comment:
I'm not a fan of this kind of relative path construction, it violates encapsulation. Instead, APIs should be added or moved to public headers.

Reply:
If you make the change to the headers as its own commit, I will cherry-pick it upstream.